### DISCUSSION TITLE
Everspring ST814 Temperature and Humidity Sensor - values no longer updated after upgrade from 8.9.0 to 8.10.1

### DISCUSSION THREAD

**Discussion by 999LV** [zwave-js/zwave-js-ui#3002] [Q&A]
Hi, I did notice that values (temp & humidity) of that (old) device are no longer updated after the UI upgrade to the last version (running zwave-js-ui as a bare metal package on a Raspberry Pi 4 64 bits PiOS). The node still is seen (wake up logs are ok) but its values are no longer sent (to domoticz in my case). Reverting to version 8.9.0 resolves the issue. Am I the only one facing this new issue ? Any pointers to help resolve ?
Thank you.

**Comment by robertsLando** [comment]
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Reply by robertsLando** [reply]
cc @AlCalzone

**Reply by AlCalzone** [reply]
@999LV have you tried re-interviewing the device?

It has previously been determined that the device doesn't need any endpoints, despite using them. Very recently, I added the ability to ignore unnecessary endpoints, and applied that to the ST814.

If that doesn't help, I will need the log @robertsLando asked for, which should include a re-interview and you triggering the missing reports somehow.

**Reply by 999LV** [reply]
@AlCalzone thanks for your suggestion... re-interviewing the device seems to have solved the issue !
